rpc/js/src/ClientChannel.ts

```diff
@@ -38,41 +52,26 @@ export class ClientChannel extends BaseChannel {
     dc.addEventListener('close', () => this.onConnectionTerminated());
   }
 
-  public transportFactory(): grpc.TransportFactory {
-    return (opts: grpc.TransportOptions) => {
-      return this.newStream(this.nextStreamID(), opts);
-    };
-  }
-
   private onConnectionTerminated() {
     // we may call this twice but we know closed will be true at this point.
     this.closeWithReason(new ConnectionClosedError('data channel closed'));
-    const err = new ConnectionClosedError('connection terminated');
     for (const streamId in this.streams) {
       const stream = this.streams[streamId]!;
-      stream.cs.closeWithRecvError(err);
+      stream.cs.closeWithRecvError();
     }
   }
 
   private onChannelMessage(event: MessageEvent<any>) {
```

Comment:
Should this be `MessageEvent<I>` (or `MessageEvent<O>` if I'm misunderstanding the terminology here) rather than `any`?

Reply:
it's neither. this is a message coming from a data channel which will be bytes